exporters/zipkin/src/main/java/io/opentelemetry/exporters/zipkin/ZipkinExporterConfiguration.java

```diff
@@ -0,0 +1,150 @@
+/*
+ * Copyright 2020, OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.opentelemetry.exporters.zipkin;
+
+import com.google.auto.value.AutoValue;
+import javax.annotation.concurrent.Immutable;
+import zipkin2.Span;
+import zipkin2.codec.BytesEncoder;
+import zipkin2.codec.SpanBytesEncoder;
+import zipkin2.reporter.Sender;
+import zipkin2.reporter.urlconnection.URLConnectionSender;
+
+/**
+ * Configurations for {@link ZipkinSpanExporter}.
+ *
+ * @since 0.4.0
+ */
+@AutoValue
+@Immutable
+public abstract class ZipkinExporterConfiguration {
+
+  ZipkinExporterConfiguration() {}
+
+  abstract String getServiceName();
+
+  abstract Sender getSender();
+
+  abstract BytesEncoder<Span> getEncoder();
+
+  /**
+   * Returns a new {@link Builder} with defaults set to an "unknown" serviceName and using the
+   * {@link SpanBytesEncoder#JSON_V2} encoder.
+   *
+   * @return a {@code Builder}.
+   * @since 0.4.0
+   */
+  public static Builder builder() {
+    return new AutoValue_ZipkinExporterConfiguration.Builder()
+        .setEncoder(SpanBytesEncoder.JSON_V2)
+        .setServiceName("unknown");
+  }
+
+  /**
+   * Builds a HTTP exporter for <a href="https://zipkin.io/zipkin-api/#/">Zipkin V2</a> format.
+   *
+   * @param endpoint The Zipkin endpoint URL, ex. "http://zipkinhost:9411/api/v2/spans".
+   * @param serviceName The serviceName with which to identify Spans. See {@link
+   *     Builder#setServiceName(String)} for details.
+   */
+  public static ZipkinExporterConfiguration create(String endpoint, String serviceName) {
+    return ZipkinExporterConfiguration.builder()
+        .setEndpoint(endpoint)
+        .setServiceName(serviceName)
+        .build();
+  }
+
+  /**
+   * Builds a HTTP exporter for <a href="https://zipkin.io/zipkin-api/#/">Zipkin V2</a> format.
+   *
+   * @param endpoint The Zipkin endpoint URL, ex. "http://zipkinhost:9411/api/v2/spans".
+   */
+  public static ZipkinExporterConfiguration create(String endpoint) {
```

Comment:
I feel it is better to have someone forced to be add their own "unknown" vs do it for them. Similar to previous comments I'm not sure the value in creating a new way to make "unknown" and I think these factory methods will do just that. personally I would delete them and just have usage similar to jaeger in README asking them to indicate a fallback (default) service name.

Reply:
remove the "unknown" from the defaults; serviceName is now required.